frontend/core-ui/src/modules/settings/general/components/GeneralSettings.tsx

```
@@ -52,8 +56,13 @@ const GeneralSettings = () => {
       const mainCurrency = configs?.find(
         (data: any) => data.code === 'mainCurrency',
       );
+
+      const timezone = configs?.find((data: any) => data.code === 'TIMEZONE');
+
       methods.setValue('dealCurrency', currencies?.value);
       methods.setValue('mainCurrency', mainCurrency?.value);
+
+      timezone && methods.setValue('TIMEZONE', timezone?.value);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)